mobile/src/screens/auth/components/sign-up-form/styles.ts

```diff
@@ -7,9 +7,10 @@ const styles = StyleSheet.create({
     height: '100%',
     width: '100%',
     paddingHorizontal: 55,
+    marginBottom: 55,
```

Comment:
Isn't it better to center all the content? (We don't know the user's screen resolution, and the layout may look different on various devices, so I suggest centering everything)

Reply:
@danilittaa if i do it so (center content), it will be just looks like this, because i was makng some improvements for sign background task with puprose to make scrollin bar at the edge< and i just moved this fixed padding from sign background. input field shouldnt have fixed width, so its look okay with screen padding
![Снимок экрана 2023-09-18 113346](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/874ce852-58df-4bd0-94c6-147f9b81ec13)

Reply:
it's better no to center screens where content may exceed screen height